templates/dns.yml.erb

```diff
@@ -3,9 +3,12 @@
 :enabled: <%= @module_enabled %>
 # valid providers:
 #   dns_dnscmd (Microsoft Windows native implementation)
+#   dns_infoblox
```

Comment:
Perhaps it's good to note that this is a plugin and not a built in.
```suggestion
#   dns_infoblox (plugin)
```

The same for powerdns and route53

Reply:
ok, done

Reply:
I don't see the change in the diff.